components/src/core/components/List/types.ts

```diff
@@ -0,0 +1,126 @@
+interface Visible {
+  visible?: boolean;
+}
+
+interface ListEmitEvent {
+  type: string;
+  identifier: string;
+}
+
+interface Tooltip {
+  tooltip?: string;
+  flow?: string;
+}
+
+interface Style {
+  [key: string]: string | undefined;
+}
+
+interface Icon {
+  iconName?: string;
+  "icon-size"?: string;
+  iconSize?: string;
```

Comment:
Why do we have icon-size and iconSize as two properties?

Reply:
removed the duplicate and proceed with camel case. made the relevant reference changes on Vue App